charts/nginx/templates/nginx-headers-configmap.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
 data:
   # Policy was generated using https://report-uri.com/home/generate
   Content-Security-Policy: "default-src 'self' *.{{ .Values.global.baseDomain }}; script-src 'unsafe-inline' 'unsafe-eval' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net cdn.astronomer.io cdn.metarouter.io cdn.segment.com www.google-analytics.com js.stripe.com widget.intercom.io js.intercomcdn.com cdn.lr-ingest.io; img-src 'self' data: *; connect-src *.{{ .Values.global.baseDomain }} wss://*.{{ .Values.global.baseDomain }} e.metarouter.io api.segment.com api.segment.io api-iam.intercom.io wss://nexus-websocket-a.intercom.io; style-src 'unsafe-inline' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net fonts.googleapis.com; frame-src js.stripe.com; font-src *.{{ .Values.global.baseDomain }} cdn.astronomer.io fonts.gstatic.com js.intercomcdn.com data:; worker-src blob:"
-  Referrer-Policy: "no-referrer"
+  Referrer-Policy: "no-referrer-when-downgrade"
```

Comment:
>The browser will not send the referrer header when navigating from HTTPS to HTTP, but will always send the full URL in the referrer header when navigating from HTTP to any origin. It doesn't matter whether the source and destination are the same site or not, only the scheme.

Reply:
Is there a "same origin" type mode we can use here instead?

Reply:
![image](https://user-images.githubusercontent.com/1374633/79003738-dab8b580-7b5b-11ea-853f-b829ddd80d07.png)

Reply:
I think that would work?

Reply:
@ashb done.
yeah tested locally, it works!